hazelcast/src/main/java/com/hazelcast/internal/util/phonehome/PhoneHomeParameterCreator.java

```diff
@@ -41,6 +41,24 @@ public PhoneHomeParameterCreator() {
         return parameters;
     }
 
+    public void addParam(PhoneHomeMetrics key, String value) {
```

Comment:
It would be much better to just call `addParam(key.getRequestParameterName(), value);` and delete the rest of this new method.